website/gatsby-config.js

```diff
@@ -274,6 +274,7 @@ module.exports = {
                 };
               }),
             title: "ChilliCream Blog",
+            output: "/rss.xml",
```

Comment:
We have now dropped the support for `atom` and `json`. I guess because this plugin is outdated and no other plugin supports the mentioned formats, right?

Reply:
As I mentioned in the description, we don't have to switch to the other plugin. I just wanted to use less forks. If there are people using these feeds, we can definitely keep it around!